### PR TITLE
Post 20.11 cleanups

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Remove deprecated and unused methods. Streamline `LabKeyJspWriter` checking.

#### Related Pull Requests
* https://github.com/LabKey/server/pull/7

#### Changes
* Remove deprecated and unused methods
* Simplify `LabKeyJspWriter` checking: remove experimental feature, statistics gathering, and logging. Just throw if a JSP attempts to print a string (in dev mode).

### CODE REVIEW

File: api/src/org/labkey/api/jsp/LabKeyJspWriter.java [hunk 1]
```diff
@@ -76,34 +36,7 @@ public void print(char[] s)
...
-            LOGSTRING.info("A JSP is printing a string!" + shortStackTrace.toString());
-        }
-
-        FILE_COUNTING_SET.add(elementWeCareAbout.getFileName(), 1);
-
-        if (ExperimentalFeatureService.get().isFeatureEnabled(EXPERIMENTAL_THROW_ON_WARNING))
-            throw new IllegalStateException("A JSP is printing a string!");
-
-        super.print(s);
+        throw new IllegalStateException("A JSP is attempting to render a string!");
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/jsp/LabKeyJspWriter.java]: Should we give a more detailed message here, perhaps with pointers to a doc page, h(), HtmlString, or similar?

**Review by labkey-jeckels — APPROVED** [review]
One suggestion, and pending getting a successful build.

File: api/src/org/labkey/api/jsp/LabKeyJspWriter.java (hunk 1 above)
Comment (labkey-susanh) [inline api/src/org/labkey/api/jsp/LabKeyJspWriter.java]: Will there be enough context here to track down where this is happening?

**Review by labkey-susanh — APPROVED** [review]

File: api/src/org/labkey/api/jsp/LabKeyJspWriter.java (hunk 1 above)
Comment (labkey-adam) [reply api/src/org/labkey/api/jsp/LabKeyJspWriter.java]: I added a link to https://www.labkey.org/Documentation/wiki-page.view?name=premServerEncoding

Comment (labkey-adam) [reply api/src/org/labkey/api/jsp/LabKeyJspWriter.java]: Yes, they'll get a stack trace in the browser and the console log that will point to the line of code in xxx_jsp.java

**Pull request merged**